app/javascript/packs/application.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from '../components/App';
```

Comment:
- [ ] Our frontend app should use both React and Redux for state management. Hence we needs a store, action and reducers in this project. A store should be provided to manage the app state :+1: